e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -317,6 +317,122 @@ describe('I18456 Built-in slash commands: common', () => {
         cy.uiWaitUntilMessagePostedIncludes(`You were added to the channel by @${user.username}`);
     });
 
+    it('MM-T660_1 /invite tests when used in DMs and GMs', () => {
+        const userDM = userGroup[2];
+
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+        cy.get('#postListContent', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible');
+
+        // # In a GM Use the /invite command to invite a channel to another channel (e.g., /invite @[channel name])
+        cy.postMessage(`/groupmsg @${userGroup[0].username} @${userGroup[1].username}`);
+        cy.postMessage(`/invite @${testChannel.name}`);
+
+        // * Error appears: "We couldn't find the user. They may have been deactivated by the System Administrator."
+        cy.uiWaitUntilMessagePostedIncludes('We couldn\'t find the user. They may have been deactivated by the System Administrator.');
+
+        cy.get('#addDirectChannel').click();
+        cy.get('#selectItems').type(`${userDM.username}`);
+        cy.findByText('Loading').should('be.visible');
+        cy.findByText('Loading').should('not.exist');
+        cy.get('#multiSelectList').findByText(`@${userDM.username}`).click();
+        cy.findByText('Go').click();
+        cy.get('#channelHeaderDropdownButton').contains(`${userDM.username}`).should('be.visible');
+
+        // # In a GM Use the /invite command to invite a channel to another channel (e.g., /invite @[channel name])
+        cy.postMessage(`/invite @${testChannel.name}`);
+
+        // * Error appears: "We couldn't find the user. They may have been deactivated by the System Administrator."
+        cy.uiWaitUntilMessagePostedIncludes('We couldn\'t find the user. They may have been deactivated by the System Administrator.');
+    });
+
+    it('MM-T660_2 /invite tests when used in DMs and GMs', () => {
+        const userDM = userGroup[2];
+        const userToInvite = userGroup[3];
+
+        cy.apiAddUserToChannel(testChannel.id, userToInvite.id);
```

Comment:
Failing on me. Should the admin logged in first before it can add user to channel?
```suggestion
        cy.apiAdminLogin();
        cy.apiAddUserToChannel(testChannel.id, userToInvite.id);
```